python/grass/jupyter/testsuite/map_test.py

```diff
@@ -101,6 +101,18 @@ def test_filename_property(self):
         # Make sure image was created
         self.assertFileExists(grass_renderer.filename)
 
+    def test_safe_file(self):
```

Comment:
Is this meant to be called `test_save_file` instead of `test_safe_file`?

Reply:
Right, although it would be great if it is safe too :-)

Reply:
I have renamed the temporary files too because they could conflict - these tests are not written in 100% isolated way which I forgot.